package.json

```
@@ -39,7 +39,7 @@
     "jest-cli": "^26.4.2",
     "js-yaml": "3.14.0",
     "lodash": "^4.17.20",
-    "sauce-testrunner-utils": "^0.3.0",
+    "sauce-testrunner-utils": "^0.4.0",
```

Would be nice to fixate the version. We don't benefit much from the caret :)

```suggestion
    "sauce-testrunner-utils": "0.4.0",
```